packages/tools/client-debugger/client-debugger/src/FluidClientDebugger.ts

```diff
@@ -34,14 +37,18 @@ import { FluidClientDebuggerProps } from "./Registry";
  * **Messages it listens for:**
  *
  * - {@link GetContainerStateMessage}: When received (if the container ID matches), the debugger will broadcast {@link ContainerStateChangeMessage}.
- *
+ * - {@link ConnectContainerMessage}: When received (if the container ID matches), the debugger will connect to the container.
+ * - {@link DisconnectContainerMessage}: When received (if the container ID matches), the debugger will disconnect from the container.
+ * - {@link CloseContainerMessage}: When received (if the container ID matches), the debugger will close the container.
  * TODO: Document others as they are added.
  *
  * **Messages it posts:**
  *
  * - {@link ContainerStateChangeMessage}: This is posted any time relevant Container state changes,
  * or when requested (via {@link GetContainerStateMessage}).
- *
+ * - {@link ConnectContainerMessage}: This is posted any time the debugger connects or disconnects from the Container.
```

Comment:
This tooling doesn't post these messages; it listens for them. You already have entries noting that we listen for them and what we do in response, so I think you can just remove these entries.

Reply:
Removed